src/server/i18n.h

```diff
@@ -65,6 +65,17 @@ class GetTranslatedString
     return expandParameterizedString(m_lang, key, params);
   }
 
+  std::pair<std::string, kainjow::mustache::basic_data<std::string>> getTranslatedPair(std::string key)
+  {
+    return {key, getTranslatedString(m_lang, key)};
+  }
+
+  std::pair<std::string, kainjow::mustache::basic_data<std::string>> getTranslatedPair(std::string key, kainjow::mustache::object params)
+  {
+    return {key, expandParameterizedString(m_lang, key, params)};
+  }
+
+
```

Comment:
1. Make the signatures consistent with those of preexisting functions.

2. It's better to introduce a separate class `GetTranslatedStringWithMsgId` with a pair of overloaded `operator()`s instead of those functions.